### PR TITLE
[AIRFLOW-5148] Adding GA and privacy notice to website

### PR DESCRIPTION
**Pull request by aijamalnk**
Make sure you have checked _all_ steps below.

### Description

Adding Google Analytics to the Website, along with a privacy notice to communicate to users about it, and about the sort of data being collected. This privacy notice is in line with other notices used in other Apache projects.

Reviewers: @kaxil

### CODE REVIEW

**Comment by mik-laj** [comment]
Travis is sad. Can you fix it? if you want I can help you tomorrow. I left my laptop charger in the office :crying_cat_face:

File: docs/index.rst [hunk 1]
```diff
@@ -94,6 +94,7 @@ Content
     lineage
     changelog
     faq
+    privacy_notice
```
Comment (kaxil) [inline docs/index.rst:97]: Added it to the toc

File: docs/templates/layout.html [hunk 2]
```diff
@@ -108,3 +113,20 @@
     }
   </style>
 {% endblock %}
+
+{% block footer %}
+{{ super() }}
+<div class="footer">This page uses <a href="https://analytics.google.com/">
+Google Analytics</a> to collect statistics. You can disable it by blocking
+the JavaScript coming from www.google-analytics.com.
```
Comment (mik-laj) [inline docs/templates/layout.html]: Can you add a link to the privacy policy here? I think it will be useful for user. It makes life easier.

Comment (kaxil) [reply docs/templates/layout.html]: Done

File: docs/templates/layout.html [hunk 3]
```diff
@@ -108,3 +113,22 @@
     }
   </style>
 {% endblock %}
+
+{% block footer %}
+{{ super() }}
+<div class="footer">This page uses <a href="https://analytics.google.com/">
+Google Analytics</a> to collect statistics. You can disable it by blocking
+the JavaScript coming from www.google-analytics.com. Check our
+<a href="privacy_notice.html">Privacy Policy</a>
```
Comment (mik-laj) [inline docs/templates/layout.html]: Does it work on subpages? 
For example: 
On this page: ``https://airflow.readthedocs.io/en/latest/howto/operator/gcp/natural_language.html``
This link will point to: ``https://airflow.readthedocs.io/en/latest/howto/operator/gcp/privay_notice.html``

Comment (kaxil) [reply docs/templates/layout.html]: You are right. it won't work. Can you fix that, please?

Comment (mik-laj) [reply docs/templates/layout.html]: I am happy that I can do it.

**Comment by mik-laj** [comment]
This message is not visible. It hides under other content.
Screenshot 2019-08-28 at 17 17 24
I will prepare fix.

**Comment by mik-laj** [comment]
Fixed
Screenshot 2019-08-28 at 17 32 42

Preview for easier review: http://equal-system.surge.sh/ (Wed Aug 28 17:36:08 CEST 2019)

**Comment by mik-laj** [comment]
Can it be merged now?  It would be nice to have new documentation with this change.

**Pull request merged**